CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## main
 
+* Add `Dropdown` component.
+
+    *Manuel Puyol*
+
+* Deprecates `DropdownMenuComponent` in favor of `Dropdown::MenuComponent`. This component should not be used standalone, being part of `Dropdown`.
```

Comment:
If `Dropdown::MenuComponent` shouldn't be used directly, perhaps we can omit this mention of it? 

```suggestion
* Deprecate `DropdownMenuComponent`.
```